src/react-components/src/__tests__/components/Hero-test.tsx

```diff
@@ -0,0 +1,92 @@
+import { expect } from "chai";
+import { stub } from "sinon";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import * as Mocha from "mocha";
+
+import Image from "../../components/01-atoms/Images/Image/Image";
+import Heading from "../../components/01-atoms/Text/Headings/Heading";
+import BodyText from "../../components/01-atoms/Text/Text/BodyText";
+import { HeroTypes } from "../../components/03-organisms/Hero/HeroTypes";
+import Hero from "../../components/03-organisms/Hero/Hero";
+
+describe("Hero Test", () => {
+
+  let blurCallback;
+  let changeCallback;
+  let submitCallback;
+
+  beforeEach(() => {
+    changeCallback = stub();
+    blurCallback = stub();
+  });
+
+  it("Generates a Hero with a background image", () => {
+    let wrapper = Enzyme.shallow(<Hero
+      heroType={HeroTypes.Primary}
+      heading={<Heading
+        level={1}
+        id={"1"}
+        text={"Hero Primary"}
+        blockName={"hero"}
+      />}
+      backgroundImageSrc="https://placeimg.com/1600/800/arch">
+    ></Hero>);
+    expect(wrapper.find(".hero")).to.have.lengthOf(1);
```

Comment:
I would test that you can find an image in this DOM since the description says it has a background image.

Reply:
Yeah, I was struggling with it and left it to solve later... and obviously forgot about it. Figured it out and updated it